internal/datacoord/import_scheduler.go

```diff
@@ -286,6 +289,11 @@ func (s *importScheduler) processInProgressImport(task ImportTask) {
 			log.Warn("update import segment rows failed", WrapTaskLog(task, zap.Error(err))...)
 			return
 		}
+		metrics.DataCoordBulkVectors.WithLabelValues(
+			strconv.FormatInt(paramtable.GetNodeID(), 10),
```

Comment:
Why is the nodeID needed?

Reply:
This metric is reported by datanode in the first design, so nodeID is needed. Forgot to remove this label when I move this to datacoord.